src/openrct2/park/Legacy.cpp

```diff
@@ -2335,6 +2335,21 @@ static bool ConvertPeepAnimationType(TPeepType* peep, AnimObjectConversionTable&
     auto& conversion = table[legacyPAG];
     peep->AnimationObjectIndex = conversion.first;
     peep->AnimationGroup = static_cast<PeepAnimationGroup>(conversion.second);
+
+    if (!peep->template Is<Staff>())
```

Comment:
NB: because the `peep` variable here is of type `TPeepType*` we have to explicitly invoke `template` to get this to parse.